tests/subsys/llext/simple/testcase.yaml

```diff
@@ -32,17 +34,22 @@ tests:
   # Run the suite with all combinations of core Kconfig options for the llext
   # subsystem (storage type, ELF type, MPU/MMU etc)
   llext.simple.readonly:
-    arch_allow: arm # Xtensa needs writable storage
-    filter: not CONFIG_MPU and not CONFIG_MMU and not CONFIG_SOC_SERIES_S32ZE
+    arch_allow: arm64 arm # Xtensa needs writable storage
+    filter: not CONFIG_MPU and not CONFIG_SOC_SERIES_S32ZE
+    integration_platforms:
+      - qemu_cortex_a53         # ARM Cortex-A53 (ARMv8-A ISA)
```

Comment:
The current `readonly` test intentionally avoids all memory protection issues so that they can be checked separately, and I'd rather not change that. 
I suggest we add a specific test for targets that strictly need MMU support, as it was made clear in the linked PR discussion:
```
llext.simple.readonly_mmu:
    arch_allow: arm64
    filter: CONFIG_MMU
    integration_platforms:
      - qemu_cortex_a53         # ARM Cortex-A53 (ARMv8-A ISA)
 ```
 @teburd what do you think?

Reply:
seems reasonable to me